M2/Macaulay2/c/Makefile.in

```diff
@@ -6,10 +6,6 @@ endif
 all: scc1 scc-core.o
 include ../../include/config.Makefile
 
-# In this directory, we compile "scc1", the D to C translator, which we want to run
-# on the "build" machine, in the sense of autoconf.  So we do not want the cross-compiling compiler:
```

Comment:
I'm fine with the CMake changes, however, I don't know why you'd remove a comment like this, especially since as far as I can tell it's not spelled out anywhere else.

Reply:
From what I could tell, it was describing the use of the `CC_FOR_BUILD` macro, which hasn't been set since we stopped calling `AX_PROG_CC_FOR_BUILD` a few years ago.

Reply:
M2 isn't cross-compiled anymore, but the information about scc1 remains useful.

Reply:
Gotcha -- I'll restore that